### PR TITLE
Feature/use content service for insurable limits and terms

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/pages/OfferNew/utils.ts [hunk 1]
```diff
@@ -35,6 +39,18 @@ export const getOfferData = (quoteBundle: QuoteBundle): OfferData => {
         currentInsurer: bundleQuote.currentInsurer,
         contractType: bundleQuote.typeOfContract,
         perils: bundleQuote.perils,
+        insurableLimits: new Map(
+          bundleQuote.insurableLimits.map((insurableLimit) => [
+            insurableLimit.type,
+            insurableLimit,
+          ]),
+        ) as ReadonlyMap<InsurableLimitType, InsurableLimit>,
```
Comment (palmenhq) [inline src/pages/OfferNew/utils.ts:47]: Really clean!

File: src/pages/OfferNew/Checkout/Sign.tsx [hunk 2]
```diff
@@ -137,12 +136,12 @@ export const Sign: React.FC<Props> = ({
...
-                ](),
-                TERMS_LINK: textKeys[
-                  getInsurancePDFTextKey(quote.contractType)
-                ](),
+                PREBUY_LINK: quote.insuranceTerms.get(
+                  InsuranceTermType.PreSaleInfo,
+                )!!.url,
+                TERMS_LINK: quote.insuranceTerms.get(
+                  InsuranceTermType.TermsAndConditions,
+                )!!.url,
```
Comment (vonElfvin) [inline src/pages/OfferNew/Checkout/Sign.tsx:144]: This will have to be updated to satisfy the Norway specific text

Comment (vonElfvin) [reply src/pages/OfferNew/Checkout/Sign.tsx:144]: `isSwedish` then one case and `isNorwegian` another

Comment (palmenhq) [reply src/pages/OfferNew/Checkout/Sign.tsx:144]: Oh no, it doesn't work or?

**Pull request merged**